prep-javascript-conditionals/main.js

```diff
@@ -0,0 +1,74 @@
+function isAdult(age) {
+  if (age > 18) {
+    return true;
+  } else if (age < 18) {
+    return false;
+  } else {
+    return console.log('Invalid age');
+  }
```

Comment:
I like how you are already thinking about error handling! However, you don't need to return a `console.log`, you can just `return` the string directly because you are already logging the result to the console

Reply:
Got it! I appreciate the tip, I can see it would be cleaner to return the string